xrpl/models/utils.py

```diff
@@ -1,10 +1,59 @@
 """Helper util functions for the models module."""
 
+import re
 from dataclasses import dataclass, is_dataclass
-from typing import Any, Dict, List, Type, TypeVar, cast
+from typing import Any, Dict, List, Optional, Pattern, Type, TypeVar, cast
+
+from typing_extensions import Final
 
 from xrpl.models.exceptions import XRPLModelException
 
+HEX_REGEX: Final[Pattern[str]] = re.compile("[a-fA-F0-9]*")
+
+# Credentials are represented in hex. Whilst they are allowed a maximum length of 64
+# bytes, every byte requires 2 hex characters for representation
+_MAX_CREDENTIAL_LENGTH: Final[int] = 64 * 2
+
+
+def _get_credential_type_error(credential_type: str) -> Optional[str]:
```

Comment:
Given that this helper function is used in other files, it should not start with a `_`

Reply:
c27ab41b9dd6194250f49c10abe82438f0882903